tests/AppBundle/Utils/CartTest.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace AppBundle\Utils;
+namespace Tests\AppBundle\Utils;
```

Comment:
I think this is useless.

Reply:
Most of the tutorials I saw where using a different namespace for tests. Not a PHP expert here, but isn't it possible that it will cause issues if some classes from the app and from the tests collide?

Reply:
Nope, because test classes are suffixed by `Test`